cypress/cypress/integration/record-queue.spec.ts

```diff
@@ -0,0 +1,67 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import {
+  cyMockDefault,
+  mockGQL,
+  cyMockFollowUpQuestions,
+} from "../support/functions";
+import { feedback as userQuestions } from "../fixtures/feedback/feedback";
+import mentor from "../fixtures/mentor/clint_new";
+
+describe("Mentor Record Queue", () => {
+  describe("Feedback Page", () => {
+    it.only("Testing record queue on feedback page", () => {
```

Comment:
remove this test since it doesn't test anything

Reply:
I may still need this for the next task, can I just comment it out? @aaronshiel

Reply:
yeah np